pom.xml

```diff
@@ -52,6 +52,7 @@
     <spring.version>4.1.4.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-transform.version>4.4.0</fcrepo-transform.version>
+    <fcrepo-java-client.version>4.4.1-SNAPSHOT</fcrepo-java-client.version>
```

Comment:
Should this be 0.2.0-SNAPSHOT instead of 4.4.1-SNAPSHOT?
https://github.com/fcrepo4-exts/fcrepo-java-client/blob/master/pom.xml#L16